src/cave.cc

```diff
@@ -1058,37 +1023,6 @@ static void map_info(int y, int x, byte *ap, char *cp)
 
 	/**** Layer 2 -- Objects ****/
 
-	if (feat != FEAT_MON_TRAP)
```

Comment:
This isn't right. This block of code should *always* run now. (Note that it's checking that the tile *isn't* a trap... which is always the case once traps are removed.)